src/routes.js

```diff
@@ -107,6 +107,22 @@ function buildRoutes() {
       </Route>
     );
   });
+
+  ["en-US", "es"].forEach(function(locale) {
```

Comment:
I didn't think this would work as well as it did.

Basically I was able create a new set of locales just for the new routes.

Reply:
so /de/buyers-guide doesn't exist.

Reply:
Nice, so what will I see with a German browser, it will redirect to /en-US/buyers-guide?

Reply:
That is correct 🚀

Reply:
or es if that's your locale set in the browser.